packages/design/sass/styles.scss

```diff
@@ -138,7 +138,7 @@ main {
 }
 
 .usa-table {
-  width: 100%;
+  width: 10 0%;
 
```

Comment:
Hey Jason. There's a space in the 100%. Is that intentional?

Reply:
No that's an error. .Thank you for catching that!

Reply:
i'll make a fix and push it the update. Thx!